src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdb/MethodBody.cs

```diff
@@ -37,5 +36,16 @@ public BlobHandle SequencePoints
                 return _reader.MethodBodyTable.GetSequencePoints(Handle);
             }
         }
+
+        /// <summary>
+        /// Returns local signature handle.
+        /// </summary>
+        public StandaloneSignatureHandle LocalSignature
+        {
+            get
+            {
+                return StandaloneSignatureHandle.FromRowId(_reader.GetBlobReader(SequencePoints).ReadCompressedInteger());
+            }
+        }
```

Comment:
Not directly related to the change or line, but this struct should be renamed. It conflicts with System.Reflection.MethodBody and also ends up looking very strange when there is also a S.R.M.MethodBodyBlock with the IL. We should convey that this has the debug info somehow in the name.

Also, it seems a bit odd that SequencePoints starts with the LocalSignature. Should we give the Blob a more opaque name?